app/assets/javascripts/osm.js.erb

```diff
@@ -10,7 +10,7 @@ OSM = {
   MAX_REQUEST_AREA:        <%= Settings.max_request_area.to_json %>,
   SERVER_PROTOCOL:         <%= Settings.server_protocol.to_json %>,
   SERVER_URL:              <%= Settings.server_url.to_json %>,
-  API_VERSION:             <%= Settings.api_version.to_json %>,
+  API_VERSION:             <%= Settings.api_versions.min_by(&:to_f).to_json %>,
```

Comment:
Why is this value set to "min_by", and what are the implications of it? Does `&:to_f` play nice with semver (e.g. 1.2.0)?

Reply:
It's mainly just min_by to keep everything working for now, since it will pick 0.6 unless the site operator chooses to only deploy 0.7. 

It's used for the bits of the website that talk to the API, like notes and changeset comments. Since there's no changes yet, it's more of a "pick either" situation.

On a wider point, I'd rather work on refactoring those bits of the site to just be regular webpages like diary entry comments, but that's a different project!